tox.ini

```diff
@@ -10,5 +10,5 @@ setenv =
 deps =
     .[all]
 commands =
-    py.test --cov={envsitepackagesdir}/ebb_lint --cov-report= -rxs [] {envsitepackagesdir}/ebb_lint
+    py.test {posargs} --cov={envsitepackagesdir}/ebb_lint --cov-report= -rxs [] {envsitepackagesdir}/ebb_lint
```

Comment:
Cool, thanks. Also I'm completely baffled by pytest's decision to _remove_ the long version of `-r`. What the hell? It's also not a treat to find it in the docs. Ugh. I scowl at it pretty mightily because I feel like the consensus for command-line tools is "`-rxs` is equivalent to `-r -x -s`," and this is gratuitously counter to that. The deprecated version, `--report=xs`, would be _so much clearer._ Especially because pytest _has_ a valid `-s`. What the hecky darn are they doing?